### PR TITLE
Feature/toast notification Component

### PR DESCRIPTION
**Pull request by SakshiGrg**

### CODE REVIEW

File: lib/components/molecules/Toast/Toast.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+  const close = () => {
+    setshowToast(false);
+  };
+
+  return (
+    <>
+      {showToast && (
+        <div className={className} showToast={showToast} autoDismissTimeout={autoDismissTimeout}>
+          <button className="toast_close" onClick={() => close()}>
+            &times;
```
Comment (vinodloha) [inline lib/components/molecules/Toast/Toast.js:43]: Use an Icon or have a way to define it from outside

File: lib/components/molecules/Toast/Toast.js [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+   * This function closes the Toast when close button inside the Toast gets clicked
+   */
+  const close = () => {
+    setshowToast(false);
+  };
+
+  return (
+    <>
+      {showToast && (
+        <div className={className} showToast={showToast} autoDismissTimeout={autoDismissTimeout}>
```
Comment (vinodloha) [inline lib/components/molecules/Toast/Toast.js:41]: How do you make it accessibility complient?

Comment (vinodloha) [reply lib/components/molecules/Toast/Toast.js:41]: why you need to set showToast={showToast} autoDismissTimeout={autoDismissTimeout}

File: lib/components/molecules/Toast/Toast.js [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+          <button className="toast_close" onClick={() => close()}>
+            &times;
+          </button>
+          <div className="toast_text">{children}</div>
```
Comment (vinodloha) [inline lib/components/molecules/Toast/Toast.js:45]: toast_close and toast_text poor class naming conention

File: lib/components/molecules/Toast/Toast.js [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+  }, []);
+
+  /*
+   * This function closes the Toast when close button inside the Toast gets clicked
+   */
+  const close = () => {
+    setshowToast(false);
+  };
+
+  return (
```
Comment (vinodloha) [inline lib/components/molecules/Toast/Toast.js:38]: functional scope is still pending

File: lib/components/molecules/Toast/Toast.story.js [hunk 5]
```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+// import { action } from '@storybook/addon-actions';
+import defaultConfig from './Toast.mock';
+
+// Import Styled Component to showcase variations
+import Toast, { ToastVanilla } from '.';
+
+// TODO: indentation is not working fine in editor. Need to check for .hbs files
+// $FlowFixMe eslint-disable-next-line extra-rules/no-commented-out-code
```
Comment (vinodloha) [inline lib/components/molecules/Toast/Toast.story.js:10]: remove // $FlowFixMe eslint-disable-next-line extra-rules

File: lib/components/molecules/Toast/Toast.story.js [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+      Wow so easy!
+    </Toast>
+  ))
+  .add('Toast Info', () => (
+    <Toast isVisible={true} variation="info">
+      <img src="/images/warning.png" alt="message icon" width="25px" height="25px" />
+      Wow so easy!
+    </Toast>
+  ))
+  .add('CTA Knobs', () => (
```
Comment (vinodloha) [inline lib/components/molecules/Toast/Toast.story.js:25]: Fix names

File: lib/components/molecules/Toast/Toast.style.js [hunk 7]
```diff
@@ -0,0 +1,48 @@
+import { css } from 'styled-components';
+
+const ToastStyles = css`
+  z-index: 1;
+  border: none;
+  display: flex;
+  align-items: center;
+  border-radius: 0;
+  position: relative;
+  min-height: 80px;
```
Comment (vinodloha) [inline lib/components/molecules/Toast/Toast.style.js:10]: Why hard coded values ?

File: lib/components/molecules/Toast/Toast.style.js [hunk 8]
```diff
@@ -0,0 +1,48 @@
+import { css } from 'styled-components';
+
+const ToastStyles = css`
+  z-index: 1;
+  border: none;
+  display: flex;
+  align-items: center;
+  border-radius: 0;
```
Comment (vinodloha) [inline lib/components/molecules/Toast/Toast.style.js:8]: Unnecessary values border-radius: 0;

File: lib/components/molecules/Toast/Toast.style.js [hunk 9]
```diff
@@ -0,0 +1,48 @@
...
+  position: relative;
+  min-height: 80px;
+  min-width: 400px;
+  top: 30px;
+  right: 30px;
+  background-image: url('.../static/images/warning.png');
```
Comment (vinodloha) [inline lib/components/molecules/Toast/Toast.style.js:14]: Don't use background image please

File: lib/components/molecules/Toast/Toast.style.js [hunk 10]
```diff
@@ -0,0 +1,48 @@
...
+  position: fixed;
+  .toast_text {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    width: 250px;
+    ${props => `${props.theme.typography.FONT_H2}`};
+  }
+  .toast_close {
+    position: absolute;
```
Comment (vinodloha) [inline lib/components/molecules/Toast/Toast.style.js:36]: Lets reduce this unnecessary CSS

File: lib/components/molecules/Toast/Toast.style.js [hunk 11]
```diff
@@ -0,0 +1,48 @@
...
+  ${props =>
+    props.variation === `info`
+      ? `
+    ${props.theme.colors.CTA_SECONDARY};
+  `
+      : ``};
+  ${props => (props.inheritedStyles ? props.inheritedStyles : '')};
+  position: fixed;
+  .toast_text {
+    display: flex;
```
Comment (vinodloha) [inline lib/components/molecules/Toast/Toast.style.js:29]: Don't understand the purpose of 

display: flex;
    align-items: center;
    justify-content: space-between;
    width: 250px;

File: lib/components/molecules/Toast/tests/Toast.test.js [hunk 12]
```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import Toast from '../index';
+
+describe('<Toast />', () => {
```
Comment (vinodloha) [inline lib/components/molecules/Toast/tests/Toast.test.js:6]: This is not good enough

File: lib/components/molecules/Toast/Toast.js [hunk 13]
```diff
@@ -0,0 +1,63 @@
...
+  /*
+   * showToast: state to manage the visibility of Toast
+   * default is true
+   */
+  const [showToast, setshowToast] = useState(isVisible);
+  //  console.log('test--->',);
+  useEffect(() => {
+    // ... Use hooks
+    /*
+     * This function is seeting the Toaster to load after some time interval
```
Comment (vinodloha) [inline lib/components/molecules/Toast/Toast.js:23]: Wrong comment

File: lib/components/molecules/Toast/Toast.story.js (hunk 5 above)
Comment (SakshiGrg) [reply lib/components/molecules/Toast/Toast.story.js:10]: fixed

File: lib/components/molecules/Toast/Toast.js (hunk 1 above)
Comment (SakshiGrg) [reply lib/components/molecules/Toast/Toast.js:43]: Resolved

File: lib/components/molecules/Toast/Toast.js (hunk 2 above)
Comment (SakshiGrg) [reply lib/components/molecules/Toast/Toast.js:41]: Resolved. autoDismissTimeout is dismissing the Toast after every 8seconds defined as prop using mock data

File: lib/components/molecules/Toast/Toast.js (hunk 3 above)
Comment (SakshiGrg) [reply lib/components/molecules/Toast/Toast.js:45]: Resolved

File: lib/components/molecules/Toast/Toast.js (hunk 4 above)
Comment (SakshiGrg) [reply lib/components/molecules/Toast/Toast.js:38]: Resolved

File: lib/components/molecules/Toast/Toast.story.js (hunk 6 above)
Comment (SakshiGrg) [reply lib/components/molecules/Toast/Toast.story.js:25]: Resolved

File: lib/components/molecules/Toast/Toast.style.js (hunk 9 above)
Comment (SakshiGrg) [reply lib/components/molecules/Toast/Toast.style.js:14]: Fixed

File: lib/components/molecules/Toast/Toast.style.js (hunk 11 above)
Comment (SakshiGrg) [reply lib/components/molecules/Toast/Toast.style.js:29]: Fixed

File: lib/components/molecules/Toast/tests/Toast.test.js (hunk 12 above)
Comment (SakshiGrg) [reply lib/components/molecules/Toast/tests/Toast.test.js:6]: Fixed

File: lib/components/molecules/Toast/Toast.js (hunk 13 above)
Comment (SakshiGrg) [reply lib/components/molecules/Toast/Toast.js:23]: Fixed

File: lib/components/molecules/Toast/Toast.style.js (hunk 7 above)
Comment (SakshiGrg) [reply lib/components/molecules/Toast/Toast.style.js:10]: Fixed

File: lib/components/molecules/Toast/Toast.style.js (hunk 8 above)
Comment (SakshiGrg) [reply lib/components/molecules/Toast/Toast.style.js:8]: Resolved

File: lib/components/molecules/Toast/Toast.style.js (hunk 10 above)
Comment (SakshiGrg) [reply lib/components/molecules/Toast/Toast.style.js:36]: fixed

**Pull request closed without merging**